Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -140,4 +141,14 @@ void PrefetchCount(EndpointConfiguration endpointConfiguration)
         #endregion
     }
 
+    void SetClientCertificates(EndpointConfiguration endpointConfiguration)
+    {
+        #region rabbitmq-config-client-certificates
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.SetClientCertificates(new X509CertificateCollection());
```

Comment:
@bording it's a good question. I wonder if it would be OK to declare a collection before the snippet region, and use it within the snippet region.

@SimonCropp could you advise?